src/index.tsx

```diff
@@ -4,19 +4,19 @@ import { HashRouter as Router } from 'react-router-dom';
 
 import { store } from './app/store';
 import { App } from './App';
-import { UsersProvider } from './components/UsersContext';
+// import { UsersProvider } from './components/UsersContext';
 
 const container = document.getElementById('root') as HTMLElement;
 const root = createRoot(container);
 
 const Root = () => (
   <Provider store={store}>
     {/* Remove UsersProvider when you move users to Redux store */}
-    <UsersProvider>
-      <Router>
-        <App />
-      </Router>
-    </UsersProvider>
+    {/* <UsersProvider> */}
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming